### ISSUE TITLE
Import OS Module - Github Repo

### ISSUE THREAD

**Issue by jairamjidgekar** [harshiitsingh/Amazing-Stuff-with-Python#1]
Hi Harshit. 

Hope you are doing good. 

I would like to know, which module I have to use to create folder in the GITHUB Repo using Python script. 

I am trying to execute python script using the GH Actions. In the py script, if the folder doesn't exist, I would like to create one dynamically using the python script. 

I am running the script on ubuntu. 

I have added below code. But that didn't help. 

Code snippet:

```
import os

#Path references
static_file_path='Data/Reference_Data/'
download_file_path='Data/Extracted_Data/Ministries/'

#Creating directories
os.makedirs(download_file_path) 
os.makedirs(static_file_path) 

```

Error in Actions after executing:

```
ERROR: Could not find a version that satisfies the requirement os (from versions: none)
ERROR: No matching distribution found for os
Error: Process completed with exit code 1.

```
Any help in this regard?

Thanks,
Jairam P.

**Comment by harshiitsingh** [comment]
Hi Jairam,
Sorry for the late reply. 
Is your issue resolved? If not, can you please provide the complete script of your GitHub actions? So that I can look into it.

**Comment by jairamjidgekar** [comment]
Hey Harshit, 

Thank you, Yes, the issue is resolved. 
I had added "os" module in the requirements.txt file. During installation, it was failing (in GH Actions).  
I simply removed it from requirements.txt file and kept it as is in the .py script. 
It worked without any glitch (Not sure why it failed when it was added in the requirements.txt file. I assume, os is preconfigured in GH Actions and it worked). 

Thank you, 
Jairam P.
